src/app/shared/redirectHelper.js

```diff
@@ -1,8 +1,8 @@
 const { logError } = require("./loggerHelper");
 
 module.exports = {
-  saveSessionAndRedirect: async (req, res, redirectUrl) => {
-    await req.session.save(function (err) {
```

Comment:
Typescript would have spotted this immediately!

I'm making good progress on running TS/JS interop, so watch this space...